test/pylib/scylla_cluster.py

```diff
@@ -25,7 +25,7 @@
 from test.pylib.host_registry import Host, HostRegistry
 from test.pylib.pool import Pool
 from test.pylib.rest_client import ScyllaRESTAPIClient, HTTPError
-from test.pylib.util import LogPrefixAdapter
+from test.pylib.util import LogPrefixAdapter, read_last_line
```

Comment:
I must be missing something, but the patch title should be "copy ...", not  move, because the `read_last_line` used in this file isn't removed and replaced with its new incarnation in util.py :confused:

Reply:
fixed